Marlin/Conditionals_post.h

```diff
@@ -379,6 +379,152 @@
   #define ARRAY_BY_HOTENDS1(v1) ARRAY_BY_HOTENDS(v1, v1, v1, v1, v1, v1)
 
   /**
+   * X_DUAL_ENDSTOPS endstop reassignment
+   */
+  #if ENABLED(X_DUAL_ENDSTOPS)
+    #define _XMIN_ 100
+    #define _YMIN_ 200
+    #define _ZMIN_ 300
+    #define _XMAX_ 101
+    #define _YMAX_ 201
+    #define _ZMAX_ 301
```

Comment:
Instead of defining these multiple times — once per dual-endstop — they should only be defined once, above these items.